getting_started_moa/creating-first-moa-cluster.adoc

```diff
@@ -0,0 +1,46 @@
+:context: creating-first-moa-cluster
+[id="creating-first-moa-cluster"]
+= Getting started with {product-title}
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+This guide walks through setting up your first {product-title} cluster using `moactl`.
```

Comment:
Do we want to refer to this as a Tutorial vs Guide? To me a guide encompasses all of the product documentation, but in a Getting Started format it's more of  Tutorial structure. Just a thought. I'm fine with Guide if that's consistent with other RH docs.